### DISCUSSION TITLE
How add new specific POI inside Cruiser

### DISCUSSION THREAD

**Discussion by jclaudej** [devemux86/cruiser#116] [Q&A] [answered]
Hi,

How add new specific POI inside Cruiser, for example motorbike brand garages, danger zone....
I think that is possible with (POI import) but what is the format of these POI ?

Thank  a lot for your answer 

PS : I love the new design, because now is possible to manage this appliaction with my gloves

Have nice day

Jean-Claude

**Comment by devemux86** [comment, marked as answer]
> How add new specific POI inside Cruiser, for example motorbike brand garages, danger zone....

Do you mean adding POI as permanent (orange) favorites?
This is possible with menu "Favorites | Import" with gpx files having waypoints.

Favorites will be improved a lot in the future with the favorite categories:
- https://github.com/devemux86/cruiser/discussions/45

Or you can import POI as temporary (blue) overlays with menu "Routing | Import".

> I love the new design, because now is possible to manage this application with my gloves

Thanks, that was its purpose, to make the app easier to use!
(and more interface design improvements are planned)

If you have suggestions for any improvements, please let me know here:
- https://github.com/devemux86/cruiser/discussions/82
